Update `Kudosu` (#7490)
ppy/osu-wiki: wiki/Modding/Kudosu/en.md

**Kudosu** is a reward system for [modding](/wiki/Modding). It awards points for successful mod posts in [beatmap discussions](/wiki/Beatmap_discussion). Accumulated points can be seen on a user's profile page.

> Kudosu is awarded to mod posts that are helpful and point out useful suggestions for the mapset. If a mapper believes that a mod post is useful and addresses a valid issue, they can click the thumbs up button and award kudosu to the user who posted the mod. 

Reviewer: ```suggestion
Kudosu is awarded to mod posts that are helpful and point out useful suggestions for the beatmap. If a mapper believes that a mod post is useful and addresses a valid issue, they can click the thumbs up button and award kudosu to the user who posted the mod. 
```

> In [forum modding](/wiki/Modding/Forum_modding), kudosu was exchangeable for [star priority](/wiki/Modding/Star_priority).

Reviewer: ```suggestion
In [forum modding](/wiki/Modding/Forum_modding), kudosu used to be exchangeable for [star priority](/wiki/Modding/Star_priority).
```

Reviewer 2: curious what u feel the difference is, these read identically to me

Reviewer: slightly emphasises that it's not the case anymore

> Nowadays, kudosu is used for the evaluation of a user's capability to become a [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators). A user must have 200 kudosu in osu! or 150 kudosu in osu!taiko, osu!catch or osu!mania in order to apply.

Reviewer: ```suggestion
Nowadays, kudosu is used for the evaluation of a user's capability to become a [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators). A user must have 200 kudosu in osu! or 150 kudosu in osu!taiko, osu!catch, or osu!mania in order to apply.
```
